src/cljs/ataru/application_common/application_field_common.cljs

```diff
@@ -25,6 +26,59 @@
 (defonce builder (new HtmlSanitizer.Builder))
 (defonce html-sanitizer (.build builder))
 
+(defonce person-block-start-keyword
+  [\# \I \F \_ \P \E \R \S \O \N])
+
+(defonce person-block-end-keyword
+  [\# \F \I \_ \P \E \R \S \O \N])
+
+(defn- person-block-start
+  [[trimmed personblock? text state]]
+  (if (and (not personblock?)
+           (= person-block-start-keyword (take (count person-block-start-keyword) trimmed)))
```

Comment:
Jos tän tarkistuksen tekis `string/includes?` avulla, niin `person-block-start-keyword` ja `person-block-end-keyword` vois olla merkkijonoja ja trimmaukselle ei pitäis olla tarvetta.